kubejs/server_scripts/tags.js

```diff
@@ -42,5 +42,68 @@ ServerEvents.tags("item", e => {
 ServerEvents.tags("block", e => {
   e.remove("minecraft:needs_iron_tool", ["create:zinc_ore", "create:deepslate_zinc_ore"]);
 
-  e.add("minecraft:needs_stone_tool", ["create:zinc_ore", "create:deepslate_zinc_ore"]);
+  e.add("minecraft:needs_stone_tool", [
+    "create:zinc_ore",
+    "create:deepslate_zinc_ore",
+    "ancient_elements:lead_ore",
+    "ancient_elements:block_of_raw_lead"
+  ]);
+
+  e.add("minecraft:needs_iron_tool", [
+    "ancient_elements:jungle_steel_ore",
+    "ancient_elements:blockof_raw_jungle_steel",
+    "ancient_elements:steel_ore",
+    "ancient_elements:block_of_raw_steel",
+    "ancient_elements:palladium_ore",
+    "ancient_elements:block_of_raw_palladium"
+  ]);
+
+  e.add("minecraft:needs_diamond_tool", [
+    "ancient_elements:steel_ender_ore",
+    "ancient_elements:block_of_raw_ender_steel",
+    "ancient_elements:titanium_ore",
+    "ancient_elements:block_of_raw_titanium",
+    "ancient_elements:nether_steel_ore",
+    "ancient_elements:block_of_raw_nether_steel",
+    "ancient_elements:abyssal_debris",
+    "ancient_elements:ancient_metal_debris"
+  ]);
+
+  e.add("forge:needs_netherite_tool", [
+    "ancient_elements:frost_ore",
+    "ancient_elements:block_of_frost_shard",
+    "ancient_elements:meteorite_ore",
+    "ancient_elements:block_of_raw_meteorite",
+    "ancient_elements:void_steel_ore",
+    "ancient_elements:block_of_raw_void_steel"
+  ]);
+
+  e.add("hltweaker:needs_ancient_elements_tier_five_tool", [
```

Comment:
Tags for blocks to display correct mining level in Jade https://github.com/yzl210/HarvestLevelTweaker?tab=readme-ov-file#set-block-harvest-level